kolibri/core/assets/src/views/userAccounts/PasswordTextbox.vue

```diff
@@ -122,11 +122,14 @@
         this.$refs.password.focus();
       },
       checkErrorsAndSubmit(e) {
-        if (this.valid) {
-          this.$emit('submitNewPassword');
-        } else {
-          // Blurring will cause validation errors to show if needed
-          e.target.blur();
+        if (e.key === 'Enter') {
```

Comment:
I don't quite understand this. If `checkErrorsAndSubmit` is triggered by the the `@keydown.enter="checkErrorsAndSubmit"` event handler, why do you need to check in the event key is 'Enter'?  https://vuejs.org/guide/essentials/event-handling.html#key-modifiers

Reply:
Sorry, you're right. It was a mistake. I will fix it.

Reply:
@jredrejo I have corrected it.